domains/authorization/Authorization.API/API/ValueObjects/IdpUserId.cs

```diff
@@ -6,11 +6,16 @@ public class IdpUserId : ValueObject
 {
     public Guid Value { get; }
 
-    public IdpUserId(Guid value)
+    private IdpUserId(Guid value)
     {
         if (value == Guid.Empty)
             throw new ArgumentException("Value cannot be an empty Guid.", nameof(value));
 
         Value = value;
     }
+
+    public static IdpUserId Create(Guid value)
```

Comment:
Can the guid be all zeroes? 0000-0000-0000-0000? Should it be allowed to?

Reply:
As far I remember 0-0-0-0 guid is also empty guid. But not 100% sure.

Reply:
both empty